src/internal/m365/service/sharepoint/backup.go

```diff
@@ -140,3 +141,7 @@ func ProduceBackupCollections(
 
 	return collections, ssmb.ToReader(), canUsePreviousBackup, el.Failure()
 }
+
+func ListsMetadataFileNames() []string {
```

Comment:
Could you please leave a note that delta file is excluded since lists doesn't support delta queries?